tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc

```diff
@@ -103,6 +103,15 @@ std::pair<TfGpuId, PlatformGpuId> GetFirstValidDeviceId() {
   return std::make_pair(TfGpuId(-1), PlatformGpuId(-1));
 }
 
+// Returns false for const nodes (we intend to drop control edges from those).
+bool ShallKeepControlEdgeFrom(const Node* input_node) {
+  if (!input_node) {
+    VLOG(2) << "Node pointer is null, this should not happen";
```

Comment:
Shouldn't we use LOG(FATAL) instead then?

Reply:
Yes, we could.